src/metabase/server/middleware/security.clj

```diff
@@ -120,8 +120,8 @@
 
 (defn- embedding-app-origin-sdk
   []
-  (when (embed.settings/enable-embedding)
-    (str "localhost:* " (embed.settings/embedding-app-origin))))
+  (when (embed.settings/enable-embedding-sdk)
+    (str "localhost:* " (embed.settings/embedding-app-origins-sdk))))
```

Comment:
Hook this up with the new settings. Please note that the BE change here is the minimal version needed for me to unblock testing, and the proper version would be implemented in milestone 0.